Lib/unittest/test/testmock/testpatch.py

```diff
@@ -9,6 +9,7 @@
 from unittest.test.testmock import support
 from unittest.test.testmock.support import SomeClass, is_instance
 
+from test.test_importlib.test_namespace_pkgs import sys_modules_context
```

Comment:
`test.test_importlib.util.uncache()` is less strong than `test.test_importlib.test_namespace_pkgs.sys_modules_context()`, but I think in this case it would be enough.

Reply:
Resolved.